test/setup/index.ts

```diff
@@ -62,18 +66,23 @@ async function createEnvironment(): Promise<void> {
   await sleep(5000);
 }
 
-async function predeploy(integrationId: string): Promise<void> {
+async function predeploy(integrationId: string, namespace: string): Promise<void> {
   try {
-    const namespace = 'snyk-monitor';
-    await kubectl.createNamespace(namespace);
-
     const secretName = 'snyk-monitor';
+    console.log(`Creating namespace ${namespace} and secret ${secretName}`);
+
+    try {
+      await kubectl.createNamespace(namespace);
+    } catch (error) {
```

Comment:
Why would this fail, how can it exist already?

Reply:
For the case when we try to create the `marketplace` namespace, it already exists and it will raise an error. To avoid that I put it around the try-catch statement.